scripts/pipelines/azure-devops/templates/eks/eks-pipeline.cfg

```
@@ -9,11 +9,7 @@ commonKubernetesPath="scripts/pipelines/azure-devops/templates/common/kubernetes
 # YAML file name.
 yamlFile="eks-provisioning.yml"
 # Source branch.
-sourceBranch="feature/eks-provisioning"
-# Path to the pipelines.
-pipelinePath=".pipelines"
-# Path to the scripts.
-scriptFilePath=".pipelines/scripts"
+sourceBranch="feature/eks-provision"
```

```suggestion
sourceBranch="feature/eks-provisioning"
```

Otherwise we need to modify the guide.

---

Done